pkg/cmd/cli/cmd/newapp.go

```diff
@@ -170,6 +176,16 @@ func RunNewApplication(f *clientcmd.Factory, out io.Writer, c *cobra.Command, ar
 				hasMissingRepo = true
 				fmt.Fprintf(c.Out(), "WARNING: We created an image stream %q, but it does not look like a Docker registry has been integrated with the OpenShift server. Automatic builds and deployments depend on that integration to detect new images and will not function properly.\n", t.Name)
 			}
+		case *deployapi.DeploymentConfig:
```

Comment:
Hrm - I don't know that we need them here, especially if it's easy to get that info elsewhere.  Let's omit this for now and focus on making that easy in other places for now (ie describe dc/foo would show it).

> On Apr 19, 2015, at 8:46 PM, Fabiano Franz <notifications@github.com> wrote:
> 
> In pkg/cmd/cli/cmd/newapp.go:
> 
> > @@ -170,6 +176,16 @@ func RunNewApplication(f *clientcmd.Factory, out io.Writer, c *cobra.Command, ar
> >  				hasMissingRepo = true
> >  				fmt.Fprintf(c.Out(), "WARNING: We created an image stream %q, but it does not look like a Docker registry has been integrated with the OpenShift server. Automatic builds and deployments depend on that integration to detect new images and will not function properly.\n", t.Name)
> >  			}
> > +		case *deployapi.DeploymentConfig:
> The major reason was to print the generated parameter values (MYSQL_USER, MYSQL_PASSWORD, etc).
> 
> —
> Reply to this email directly or view it on GitHub.
>